test/unit_tests/modules/view-spec.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var jsdom = require( 'mocha-jsdom' );
+var sinon = require( 'sinon' );
+var View = exports.View = require( './../fixtures/View.js' );
+var glob = require( 'glob' );
+
+runTest();
+testAllViews();
+
+function testAllViews() {
+  glob( './src/static/js/**/*View.js', '', function ( er, files ) {
+    for ( var len = files.length, i = 0; i < len; i++ ) {
+      View = exports.View = require( '../../../' + files[i] );
+      if ( View ) {
+        runTest();
+      }
+    }
+  } );
+}
+
+function runTest() {
+
+  describe( 'View', function() {
+    var viewInstance;
+    var viewInstancewithEl;
+
+    jsdom();
+
+    beforeEach( function() {
+      viewInstance = new View();
+      viewInstancewithEl = new View( { el: document.createElement( 'div' ) } );
+    } );
+
+    context( 'has a constructor and it', function() {
+      it( 'should be passed an object', function() {
+        var constructorSpy = sinon.spy( exports, 'View' );
+        View = exports.View;
+        new View( {} ); // eslint-disable-line max-statements, no-inline-comments, max-len
+        var arg0 = constructorSpy.args && constructorSpy.args[0];
+        expect( arg0 !== null && typeof arg0 === 'object' ).to.be.true;
```

Comment:
Does this actually test that it has a constructor? Maybe that's implied through the use of arguments, but it seems like this is just checking that there's an argument and it's an object.